dev-util/ruff/ruff-0.3.0-r1.ebuild

```diff
@@ -418,51 +416,25 @@ DOCS=(
 	docs
 )
 
-src_prepare() {
-	sed -r 's:(strip[[:space:]]*=[[:space:]]*)true:\1false:' \
-		-i pyproject.toml || die
-
-	default
-
-	# python_copy_sources is called in src_compile after cargo_src_compile
-	# to avoid rebuilding rust code for each python impl
-
-	rm -rf docs/{.overrides,gitignore} || die
-}
-
 src_configure() {
 	export RUSTFLAGS="${RUSTFLAGS}"
 	cargo_src_configure
 }
 
 src_compile() {
-	cargo_src_compile
-
-	python_copy_sources
-	distutils-r1_src_compile
-
-	local solib releasedir
-	releasedir=target/$(usex 'debug' 'debug' 'release')
-
-	for solib in "target/$(usex 'debug' 'debug' 'release')"/*.so; do
-		patchelf --set-soname "${solib##*/}" "${solib}" || die
-	done
+	export CARGO_FEATURE_UNPREFIXED_MALLOC_ON_SUPPORTED_PLATFORMS=1
+	export JEMALLOC_OVERRIDE="${ESYSROOT}/usr/$(get_libdir)"/libjemalloc.so
```

Comment:
I would prefer to use `local -x` here rather than adding it to the global environment

Reply:
Done.